tests/integration/services/test_lifecycle.py

```diff
@@ -72,6 +72,7 @@ def test_package_repositories_in_overlay(new_dir, mocker, run_lifecycle):
         work_dir=work_dir,
         base_layer_dir=base_layer_dir,
     )
+    # pylint: disable=protected-access
     parts_lifecycle = lifecycle_service._lcm
```

Comment:
should this be public instead?

Reply:
Maybe? I have mixed feelings about making things public for the sake of tests
I think in this case what we want is a `project_info() -> ProjectInfo | None` method instead in `LifecycleService`